spec/models/workflow_spec.rb

```diff
@@ -98,4 +98,23 @@
       end
     end
   end
+
+  describe '#external_signal? & #external_processing?' do
+    context 'no template' do
+      it 'has no external_processing' do
+        expect(workflow.external_processing?).to be_falsy
```

Comment:
@bzwei Should this be falsey or falsy?

Reply:
https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/be-matchers#be-falsey-matcher

Reply:
should be `falsey`. Why `falsy` also works?